### PR TITLE
Update to Livewire 3

### PR DESCRIPTION
**Pull request by alexmanase**
Fixes https://github.com/spatie/laravel-dashboard/issues/112

### CODE REVIEW

File: src/Dashboard.php [hunk 1]
```diff
@@ -54,7 +54,7 @@ public function assets(): HtmlString
         }
 
         foreach ($this->inlineScripts as $inlineScript) {
-            $assets[] = "<script>{$inlineScript}</script>";
+            $assets[] = "<script>$inlineScript</script>";
         }
 
         foreach ($this->stylesheets as $stylesheet) {
```
Comment (ousid) [inline src/Dashboard.php:60]: You may need to remove The curly braces on the `{$stylesheets}` also. Don't you think?

**Review by ousid — APPROVED** [review]

File: src/Dashboard.php [hunk 2]
```diff
@@ -54,7 +54,7 @@ public function assets(): HtmlString
         }
 
         foreach ($this->inlineScripts as $inlineScript) {
-            $assets[] = "<script>{$inlineScript}</script>";
+            $assets[] = "<script>$inlineScript</script>";
         }
```
Comment (ousid) [inline src/Dashboard.php:59]: You may need to remove The curly braces on the `{$stylesheets}`  also. Don't you think?

**Comment by freekmurze** [comment]
Thank you very much! Is this ready for review? Have you tried it out yourself in the browser?

**Comment by alexmanase** [comment]
Yes. I tried the components in the browser.

**Comment by freekmurze** [comment]
Thank you!

**Comment by freekmurze** [comment]
This function does not exist anymore in Livewire 3; https://github.com/spatie/laravel-dashboard/blob/be81d9b3c3e440db5d2adff69fcaa8117db4aca1/resources/views/dashboard.blade.php#L71

I wonder how this could have worked in your browser.

**Comment by freekmurze** [comment]
I've removed the tagged release as these changes are not working and / or incomplete

**Comment by alexmanase** [comment]
Sorry. I'll fix it.

**Pull request merged**